membership_group/views/membership_group_view.xml

```diff
@@ -48,20 +48,20 @@
                         </group>
                     </group>
                     <group name="form_body" />
-
                     <notebook>
                         <page string="Membership" name="membership">
-                            <field name="membership_group_member_ids">
-                                <tree editable="top">
-                                    <field
-                                        name="partner_id"
-                                        domain="[('membership_state', 'in', ['invoiced', 'paid', 'free'])]"
-                                    />
-                                </tree>
-                            </field>
+                            <field name="membership_group_member_ids" />
                         </page>
                         <page string="Subgroups">
-                            <field name="child_ids"/>
+                            <field name="child_ids" />
+                        </page>
+                        <page string="Configuration">
+                            <group>
+                                <field name="termination_cycle" widget="boolean_toggle" />
```

Comment:
I think the termination fields are a bit bloat, also this indicates that it automatically cycles but this isn't the case, the only functionality behind this is a default value for member group lines.